DEPS

```diff
@@ -299,11 +297,14 @@ deps = {
   # WARNING: Unused Dart dependencies in the list below till "WARNING:" marker are removed automatically - see create_updated_flutter_deps.py.
 
   'engine/src/flutter/third_party/dart/third_party/binaryen/src':
-   Var('chromium_git') + '/external/github.com/WebAssembly/binaryen.git@1d2e23d5e55788091a51420ba3a9889d4efe7509',
+   Var('chromium_git') + '/external/github.com/WebAssembly/binaryen.git' + '@' + Var('dart_binaryen_rev'),
 
   'engine/src/flutter/third_party/dart/third_party/devtools':
    {'dep_type': 'cipd', 'packages': [{'package': 'dart/third_party/flutter/devtools', 'version': 'git_revision:0327830448901920f739259364c3f2f624df5a03'}]},
 
+  'engine/src/flutter/third_party/dart/third_party/perfetto/src':
```

Comment:
We should have done a reverse thing: updated Dart `DEPS` to use `flutter_git` version. https://android.googlesource.com/platform/external/perfetto seems to be dead (last commit was 7 months ago), while Flutter version is tracking GitHub repo which is where development happens.